core/src/main/scala/chisel3/Data.scala

```diff
@@ -293,6 +293,9 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
     }
   }
 
+  // Whether this node has a specified flipped direction, ignoring coercion via Input/Output
```

Comment:
this scaladoc seems a little off -- it seems you are using this to indicate whether some element in the Data is Flipped? NOt whether this data itself is Flipped?